### DISCUSSION TITLE
Funds did not arrive on ETH chain. Stuck at "Sign the claim..."

### DISCUSSION THREAD

**Discussion by ghost** [wormhole-foundation/wormhole#240] [Problems / bugs] [answered]
.

**Comment by hendrikhofstadt** [comment, marked as answer]
It looks like you didn't complete the transfer on the Ethereum side but the Wormhole correctly attested your transfer. Please follow the instructions of the UI you used or ask them for support. We don't provide support for UIs.

**Comment by allmails888** [comment]
![image](https://user-images.githubusercontent.com/17685229/126712762-bb160de0-2ede-43f5-b8e9-bce5ca6306d0.png)
user feedback from solana wormhole discord

**Comment by leoluk** [comment]
You can create issues for wormholebridge.com here: https://github.com/solana-labs/oyster/issues

**Comment by leoluk** [comment]
Did this get resolved? Your tokens aren't lost, your wallet provider simply needs to retry the submission to Ethereum.
